test/test-new-project.js

```diff
@@ -3,28 +3,33 @@ const shell = require('shelljs');
 shell.config.fatal = true;
 
 const commands = [
-    'node index.js tmp-project',
-    'node index.js tmp-project --vanilla',
-    'node index.js tmp-project --rust',
-    'node index.js tmp-project --vanilla --rust'
+    'node index.js',
+    'node index.js --vanilla',
+    'node index.js --rust',
+    'node index.js --vanilla --rust'
 ];
 
 for (let i = 0; i < commands.length; i++) {
     // remove temporary blank project
-    shell.rm('-rf', 'tmp-project');
-    // test generating new project in new dir
-    shell.exec(commands[i], { silent: true });
-    shell.cd('tmp-project');
-    shell.env.FILE = 'package.json';
-    if (!shell.test('-e', shell.env.FILE)) {
-        shell.echo(`Couldn't find ${shell.env.FILE}`);
-        shell.exit(1);
-    }
+    shell.rm('-rf', `tmp-project-${i}`);
```

Comment:
given that `i` is reused between runs – it will mean that when you run tests locally (without removing dirs) – they can be passing even after breaking changes (because files from previous run are still are there) which is undesirable